### ISSUE TITLE
Spliced and Unspliced Reads Information

### ISSUE THREAD

**Issue by cinnamon6** [williamritchie/IRFinder#176]
Hello! 
I was wondering if it is possible to retrieve the spliced and unspliced reads information (basically the numerator and denominator values using which the IRratio is calculated).

**Comment by dg520** [comment]
@cinnamon6 Yes, those values are in the IRFinder output file. Please refer to [this](https://github.com/williamritchie/IRFinder/wiki/IRFinder-Output#column-explanation-for-irfinder-ir-dirtxt-and-irfinder-ir-nondirtxt) and see how the value in Column 20 (i.e. IR ratio) is calculated from the values in Columns 8, 9, 17 and 18.

**Comment by cinnamon6** [comment]
Thank you for your reply!
Apart from the intron depth, is it possible to get the counts information at the introns as well?
